src/wazuh_db/schema_agents.sql

```diff
@@ -286,6 +286,32 @@ CREATE TABLE IF NOT EXISTS sys_groups (
     PRIMARY KEY (group_name)
 );
 
+CREATE TABLE IF NOT EXISTS sys_services (
+    scan_id INTEGER,
+    scan_time TEXT,
+    name TEXT,
+    display_name TEXT,
+    description TEXT,
+    service_type TEXT,
+    start_type TEXT,
+    state TEXT,
+    pid INTEGER,
+    ppid INTEGER,
+    binary_path TEXT,
+    load_state TEXT,
+    active_state TEXT,
+    sub_state TEXT,
+    unit_file_state TEXT,
+    status TEXT,
+    user TEXT,
+    can_stop TEXT,
+    can_reload TEXT,
+    service_exit_code INTEGER,
+    checksum TEXT NOT NULL CHECK (checksum <> ''),
+    item_id TEXT,
+    PRIMARY KEY (scan_id, name)
+);
```

Comment:
This table does not match with the one implemented by the agent team. I don't see the item_id field for example. Is this correct? because this will affect the entire logic of deltas and synchronization.